src/api/app/controllers/webui/request_controller.rb

```diff
@@ -169,10 +169,14 @@ def changerequest
           if tpkg
             target = Package.find_by_project_and_name(tprj, tpkg)
           else
-            target = Project.Project.find_by_name tprj
```

Comment:
Oh, it's from [2013](374d6bea534d4fe7ee731389d2169b0ae10fcb7a). Is this codepath used at all? It's certainly misses a test...